File: onlineexamination/src/main/java/com/cg/onlineexamination/student/dto/StudentDTO.java
package com.cg.onlineexamination.student.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class StudentDTO {

	private int studentId;
	private String userName;
	private String password;
	private String firstName;
	private String lastName;
	private String gender;
	private String emailId;
	private long mobileNo;

}

File: src/app/student-dto.ts
export class StudentDto {

    studentId:number;
	username:string;
	password:string;
	firstName:string;
	lastName:string;
	gender:string;
	emailId:string;
	mobileNo:number;


    constructor(
        studentId:number,
        username:string,
        password:string,
        firstName:string,
        lastName:string,
         gender:string,
        emailId:string,
        mobileNo:number,
    

    ){
        this.studentId = studentId;
        this.username = username;
        this.password = password;
        this.firstName = firstName;
        this.lastName = lastName;
        this.gender = gender;
        this.emailId = emailId;
        this.mobileNo = mobileNo;
    }
}
